### PR TITLE
Update to single selects, fix some questions

### PR DESCRIPTION
**Pull request by rrcobb**
* Using single-select in place of all multi-select questions
* Reword some questions so the correct answer is unambiguous

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
+( ) the functions that the browser 'knows about' by default ( ) variables and functions get 'hoisted' to the top ( ) the value of the variables defined inside a function (X) where declared variables and functions are visible
 
 ?: Variables declared with \_\_\_ are block-scoped:
 
 ( ) `var` (X) `let` ( ) `function`
 
 ?: Through the scope chain, a function:
 
-(X) has access to all variables and functions declared in its outer scope. ( ) has access to all variables and functions declared only in its inner scope. ( ) has access to all variables and functions declared in its inner and outer scopes. ( ) continues to move up the scope chain when it finds a matching identifier in the current scope.
+(X) has access to all variables and functions declared in its enclosing scope. ( ) has access to all variables and functions declared only in its inner scope. ( ) has access to variables and functions declared anywhere. ( ) continues to move up the scope chain when it finds a matching identifier in the current scope.
```
Comment (sgharms) [inline README.md:17]: I'm shaky whether we actually use "scope chain" in the docs. We might have elided those in favor of plain language. It's a decent bit of jargon, but I'd love to know whether that LG is expressed somewhere explicitly (yay, LGs coming to save us all).

Comment (sgharms) [reply README.md:17]: FWIW, I'm going to add it to the new piece I created yesterday. ;) Not sure if status quo supports tho.

**Review by sgharms — APPROVED** [review]
Seems good. Added a note that's more procedural than re: the content, but I think you're probably safe to merge as is.

File: README.md [hunk 2]
```diff
@@ -1,84 +1,90 @@
...
-(X) where declared variables and functions are visible ( ) with `const` variables ( ) only in the global context ( ) only within the function block
+( ) the functions that the browser 'knows about' by default ( ) variables and functions get 'hoisted' to the top ( ) the value of the variables defined inside a function (X) where declared variables and functions are visible
 
-?: Variables declared with ___ are block-scoped:
+?: Variables declared with \_\_\_ are block-scoped:
 
-(X) `let` ( ) `var` ( ) `function`
+( ) `var` (X) `let` ( ) `function`
 
 ?: Through the scope chain, a function:
```
Comment (rrcobb) [inline README.md]:
```suggestion
?: Through its scope, a function:
```
@sgharms Is that more clear?

File: README.md (hunk 1 above)
Comment (rrcobb) [reply README.md:17]: Added a change to just mention 'scope'. I think this quiz will come before the new piece in the chronology for students, so the jargon here shouldn't jump ahead.

**Comment by sgharms** [comment]
seems safe enough to try.

**Pull request merged**